### PR TITLE
 U4-9710 ImageCropDataSet.HasImage returns wrong value 

### PR DESCRIPTION
**Pull request by pijemcolu**
Fixed

### CODE REVIEW

File: src/Umbraco.Web/Models/ImageCropDataSet.cs [hunk 1]
```diff
@@ -76,7 +76,7 @@ public bool HasCrop(string alias)
 
         public bool HasImage()
         {
-            return string.IsNullOrEmpty(Src);
+            return ! string.IsNullOrEmpty(Src);
```
Comment (vloiuk) [inline src/Umbraco.Web/Models/ImageCropDataSet.cs]: An extra space could be removed here `! string`.

Comment (pijemcolu) [inline src/Umbraco.Web/Models/ImageCropDataSet.cs]: Correct. Should I? I just thought it's more readable that way.

Comment (vloiuk) [inline src/Umbraco.Web/Models/ImageCropDataSet.cs]: It is just coding style. Sure it will work in its current form too :) Thanks for commit man!

**Comment by clausjensen** [comment]
Thanks for the PR 😄 see you on Monday

**Comment by pijemcolu** [comment]
Best PR ever right? Even mnaged to have a discussion about it. :)

**Pull request merged**